README.md

```
@@ -1 +1,75 @@
-# FINDA_SERVER
\ No newline at end of file
+# FINDA_SERVER
+```mermaid
+erDiagram
+	VOLUNTEER{
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)